integration/client.go

```diff
@@ -41,6 +41,20 @@ func (c *HyperClient) GetPodInfo(podID string) (*types.PodInfo, error) {
 	return pod.PodInfo, nil
 }
 
+// ExitCode gets exitcode by container and tag
+func (c *HyperClient) ExitCode(container, tag string) (int32, error) {
+	request := types.ExitCodeRequest{
+		Container: container,
+		Tag:       tag,
```

Comment:
@Crazykev we can make a workaround for now: Rename `ExitCode` to `Wait`, and rename `tag` to id.